### ISSUE TITLE
plugin templates not availble when using in=APP

### ISSUE THREAD

**Issue by madsflensted** [ninenines/erlang.mk#779]
Plugins not listed in erlang.mk, does not get expanded when using `in=APP` construct

Makefile:
```
DEP_PLUGINS = cowboy
```
Run:
```
make new t=cowboy.ws n=some_ws_module in=some_app
```
results in an empty file:

```
apps/some_app/src/some_ws_module.erl
```

I suspect that some variable(s) need to be exported? when calling apps/some_app/Makefile

or maybe I am missing something?

**Comment by essen** [comment]
Can you try with `DEP_EARLY_PLUGINS = cowboy` instead of `DEP_PLUGINS = cowboy`?

**Comment by madsflensted** [comment]
Using 

```
DEP_EARLY_PLUGINS = cowboy
```
also results in empty file

**Comment by essen** [comment]
OK I think it happens because the app doesn't have `DEP_PLUGINS = cowboy`. It *shouldn't have to* but the code is missing something to make this work properly.

**Comment by madsflensted** [comment]
Yes adding `DEP_PLUGINS = cowboy` to `apps/some_app/Makefile` makes this work. I can use that workaround for now, thanks!
